cmd/login_influxdb.go

```diff
@@ -54,34 +55,34 @@ This will set the default server used when just "-o influxdb" is passed.`,
 			}
 			conf = conf.Apply(urlConf)
 		}
-		if conf.Addr == "" {
-			conf.Addr = "http://localhost:8086"
+		if conf.Addr.String == "" {
+			conf.Addr = null.StringFrom("http://localhost:8086")
```

Comment:
I think it would be a bit cleaner if those default values are defined in `NewConfig()` with `null.NewString("http://localhost:8086", false)` and `null.NewString("k6", false)` respectively.